Dockerfile

```
@@ -1,205 +1,62 @@
+
+# Copy dependency files
+COPY mix.exs mix.lock ./
+
+# Get dependencies
+RUN mix deps.get --only prod
+
+# Copy application code
+COPY . .
+
+# Compile the application
+RUN mix do compile, release
```

Hadolint (2.12.0)

[warning] Use semicolon or linefeed before 'do' (or quote to make it literal).

(SC1010)